README.md

```diff
@@ -20,24 +20,30 @@ Or install it yourself as:
 ## Configuration
 
     Twofishes.configure do |config|
-      config.base_url = 'http://mytwofishes.com:8081'
-      config.timeout = 3 # in seconds
+      config.address = '127.0.0.1:8080'
+      config.retries = 2
```

Comment:
does thrift support a timeout? if not, shall we properly deprecate this configuration option and print a warning?